app/js/views/modal.jsx

```diff
@@ -21,3 +21,10 @@ export default class Modal extends React.Component {
     );
   }
 }
+
+Modal.propTypes = {
```

Comment:
> I just hope that won't be too much troubles when creating/changing components.

Right, let's see how it goes.

> If the propTypes property of components are left as is by Babel/React maybe we can write a gulp task to remove them from the transpiled code.

Good point, we need to check, hopefully ```react``` babel preset will take care of this, but I'm not sure.